main/cloud_image_validator.py

```diff
@@ -45,6 +47,9 @@ def main(self):
             console_lib.print_divider('Running tests')
             self.run_tests_in_all_instances(instances)
         finally:
+            if self.stop_cleanup:
+                console_lib.print_divider('Cleanup stopped, press any key to continue')
```

Comment:
Maybe this divider is redundant IMO. We can just leave the input message take care of letting the user know that CIV is waiting for user interaction to continue with the cleanup (see suggestion below).